test-packages/type-tests/test/v4/filter-functions.spec.ts

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import { filterFunctions } from '@sap-cloud-sdk/core/v4';
+import { TestEntity } from '@sap-cloud-sdk/test-services/v4/test-service';
+
+// $ExpectType Filter<TestEntity, string>
+filterFunctions
+  .substring(TestEntity.STRING_PROPERTY, TestEntity.INT_16_PROPERTY)
```

Comment:
Is the substring() fulter function v4 specific? I see it also in the common filter-functions.ts

Reply:
It is not specific, but I wanted to check that one specific and one unspecific are there, so both are available in v4.